lacecore/_obj.py

```diff
@@ -0,0 +1,79 @@
+from collections import OrderedDict
+import numpy as np
+
+from ._group_map import GroupMap
+from ._mesh import Mesh
+
+ERROR_MESSAGE = "tinyobjloader library has not been installed. You will not be able to load OBJ files"
+
+try:
+    from tinyobjloader import ObjReader, ObjReaderConfig
+except Exception:  # pragma: no cover
+    ObjReader = None
+    ObjReaderConfig = None
+    import sys
+
+    print("WARNING: %s" % ERROR_MESSAGE, file=sys.stderr)
+
+
+class LoadException(Exception):
+    pass
+
+
+class ArityException(Exception):
+    pass
+
+
+def load(mesh_path, triangulate=False):
+    """
+    Load a `Mesh` from a path to an OBJ file.
+
+    Args:
+        mesh_path (str): A path to an OBJ file
+        triangulate (bool): A flag that indicates whether to triangulate the mesh on load.
+
+    Returns:
+        lacecore.Mesh: A `Mesh` instance
+    """
+    if ObjReader is None:  # pragma: no cover
+        raise Exception(ERROR_MESSAGE)
+    reader = ObjReader()
+    config = ObjReaderConfig()
+    config.triangulate = triangulate
+    success = reader.ParseFromFile(mesh_path, config)
+    if success is False:
```

Comment:
A bit more Pythonic:

```suggestion
    if not success:
```